deploy/apache/sites-available/privacyidea.conf

```diff
@@ -11,6 +11,10 @@
 		AllowOverride None
 	</Directory>
 
+        # CSP Content-Security-Policy, protect website from click-jacking attacks.
+        # Allow from self but DENY others
+        Header set Content-Security-Policy "frame-ancestors 'self';"
```

Comment:
Why do You use `frame-ancestors` here but `default-src` in the nginx config?

Reply:
you're right, should be like the apache config "frame-ancestors"

Reply:
more info:
https://docs.w3cub.com/http/headers/content-security-policy/frame-ancestors
We should additional prevent: 'unsafe-eval' or 'unsafe-inline', frame-ancestors did this as default.

todo:
[ ] test this on a loadbalancer / haproxy